### PR TITLE
Lesson 1 7

### PR DESCRIPTION
**Pull request by Maarimar**

### CODE REVIEW

File: src/TodoListItem.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 const TodoListItem = ({ todo, onRemoveTodo}) => {
   return (
-      <li key={todo.id}>{todo.title} <button onClick={()=>onRemoveTodo(todo.id)}>X</button></li>
+      <li key={todo.id}>{todo.fields.Title} <button onClick={()=>onRemoveTodo(todo.id)}>X</button></li>
```
Comment (alenamedved) [inline src/TodoListItem.js:5]: don't need to add key here, it is already been added to TodoListItem

**Review by alenamedved — COMMENTED** [review]
The code looks good, @Maarimar 
I wasn't able to fully run your app and see how it fetches cause I don't have your .env file, if you want you can Slack me, this way I'll be able to run your app

File: src/TodoListItem.js (hunk 1 above)
Comment (Maarimar) [reply src/TodoListItem.js:5]: Hi! do you mean no id in the button?

Comment (alenamedved) [reply src/TodoListItem.js:5]: Hi, I mean on 'li' item

**Pull request merged**